package.json

```diff
@@ -25,12 +26,15 @@
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
     "react-native-elements": "^1.2.7",
     "react-native-gesture-handler": "~1.6.0",
+    "react-native-ionicons": "^5.0.0-rc",
+    "react-native-paper": "^3.10.1",
     "react-native-kill-packager": "^1.0.0",
     "react-native-reanimated": "~1.7.0",
     "react-native-safe-area-context": "0.7.3",
     "react-native-screens": "~2.2.0",
     "react-native-svg": "^12.1.0",
     "react-native-swipe-gestures": "^1.0.5",
+    "react-native-vector-icons": "^6.6.0",
```

Comment:
Do we need this package if we use `'@expo/vector-icons'` ?

Reply:
It's removed